apollo-router/src/router_factory.rs

```diff
@@ -96,7 +99,9 @@ impl RouterServiceFactory for YamlRouterServiceFactory {
                             name,
                             configuration
                         );
-                        match factory.create_instance(configuration) {
+                        // expand any env variables in the config before processing.
+                        let configuration = expand_env_variables(configuration);
```

Comment:
maybe add a note to the changelog about env variables expanding in the config? That feature will help a lot!

Reply:
Fixed in [59979fa](https://github.com/apollographql/router/pull/782/commits/59979fa8621c58aa57ff29dd17b258fcebf94c5d)